wasm/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "css-inline-wasm"
+version = "0.3.0"
+authors = ["Dmitry Dygalo <dadygalo@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+name = "css_inline"
+crate-type = ["cdylib"]
+
+[dependencies.css-inline]
+path = ".."
+version = "= 0.3.3"
+default-features = false
+
+[dependencies]
+url = "2"
+wasm-bindgen = { version = "0.2", features = ["serde-serialize"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_derive = "1"
+serde_json = "1"
```

Comment:
And maybe another small thing:

```toml
[profile.release]
codegen-units = 1
lto = "fat"
```

To have some performance & size optimizations.